poster/poster.tex

```diff
@@ -259,8 +265,8 @@ \section*{DQInspect}
 \end{center}
 
 \para
-{\em DQInspect} (see Fig. 4) is used to inspect the associated DQ array of a
-given image.
+{\em DQInspect} (see Fig. 4) is used to visualize the associated DQ (Data
+Quality) array stored as an HDU within an image.
```

Comment:
An explanation of what DQ means was already done under "BackgroundSub", is this repetition really necessary?

Reply:
I'm fine with it as is if you want to leave it out.